src/main/java/ru/yandex/qatools/ashot/shooting/ViewportPastingDecorator.java

```diff
@@ -43,6 +43,18 @@ public BufferedImage getScreenshot(WebDriver wd, Set<Coords> coordsSet) {
         int viewportHeight = getWindowHeight(wd);
         shootingArea = getShootingCoords(coordsSet, pageWidth, pageHeight, viewportHeight);
 
+        if (pageHeight > 2000) {
+            System.out.printf("pageHeightが %d なのは何かがおかしいです。\n", pageHeight);
+        }
+        System.out.printf("pageWidth = %d,pageHeight = %d,viewportHeight = %d, shootingArea.height = %d\n",
+                pageWidth, pageHeight, viewportHeight, shootingArea.height);
+        if (coordsSet == null)  {
+            System.out.println("coordsSet is null");
+        } else {
+            Coords c = Coords.unity(coordsSet);
+            System.out.printf("Coords.unity(coordsSet) = (%d,%d,%d,%d)\n", c.x, c.y, c.width, c.height);
+        }
+
```

Comment:
The code above should be removed... because it had wrongly committed from my debugging code

Reply:
wrong commit removed